RELEASE_NOTES.rst

```diff
@@ -6450,10 +6450,10 @@ The following table shows changes in import paths.
      - ``airflow.sensors.base.BaseSensorOperator``
    * - ``airflow.sensors.date_time_sensor.DateTimeSensor``
      - ``airflow.sensors.date_time.DateTimeSensor``
-   * - ``airflow.sensors.external_task_sensor.ExternalTaskMarker``
-     - ``airflow.sensors.external_task.ExternalTaskMarker``
-   * - ``airflow.sensors.external_task_sensor.ExternalTaskSensor``
-     - ``airflow.sensors.external_task.ExternalTaskSensor``
+   * - ``airflow.providers.standard.sensors.external_task_sensor.ExternalTaskMarker``
```

Comment:
This should nt be changed - those are old release notes and they should remain as they were.